src/time_bucket.h

```diff
@@ -10,6 +10,10 @@
 
 #include "export.h"
 
+#define TIME_BUCKET_NG_DEFAULT_ORIGIN_TIMETZ "2000-01-01 00:00:00+01"
+#define TIME_BUCKET_NG_DEFAULT_ORIGIN_TIME "2000-01-01 00:00:00"
```

Comment:
```suggestion
#define TIME_BUCKET_NG_DEFAULT_ORIGIN_TIMESTAMPTZ "2000-01-01 00:00:00+01"
#define TIME_BUCKET_NG_DEFAULT_ORIGIN_TIMESTAMP "2000-01-01 00:00:00"
```
Just to don't make confusion with `TIME` datatype.